bndtools.m2e/_plugin.xml

```diff
@@ -52,6 +52,7 @@
 							<or>
 								<test property="org.eclipse.jdt.core.hasTypeOnClasspath" value="org.junit.Test" />
 								<test property="org.eclipse.jdt.core.hasTypeOnClasspath" value="junit.framework.Test" />
+								<test property="org.eclipse.jdt.core.hasTypeOnClasspath" value="org.junit.jupiter.api.Test" />
```

Comment:
Need more junit 5 tests in `<or>` as in the above comment.

Reply:
No I don't think so.